motor_control/throttle/base_throttle.h

```diff
@@ -6,11 +6,12 @@ class Throttle
 public:
     Throttle(const int &pin);
 
-    virtual void compute_motor_value() = 0; // this needs to be implemented differently for variable/button throttle
+    virtual int compute_motor_value() = 0; // this needs to be implemented differently for variable/button throttle
```

Comment:
The point of this function isn't really to return the motor value. This function should be updating the `m_motor_value` data member internally.